test/browser/step_definitions/details.js

```diff
@@ -1,18 +1,23 @@
 const { Given, When, Then } = require("@cucumber/cucumber");
 
-const { DetailsPage } = require("../pages");
+const { RelyingPartyPage } = require("../pages");
+const { expect } = require("chai");
```

Comment:
Do we need this if its in global test scope?

Reply:
Probably not. I might come back to this when I clean it up.